src/librustc/session/config.rs

```diff
@@ -1410,6 +1410,9 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     merge_functions: Option<MergeFunctions> = (None, parse_merge_functions, [TRACKED],
         "control the operation of the MergeFunctions LLVM pass, taking
          the same values as the target option of the same name"),
+    internal_lints: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
Instead of adding a new flag we could just reuse the -Zunstable-features flag. If the lints are allow by default, we can add deny attributes to crates in steps, and even with cfg_attr to only turn them on after stage0. That will get us around having to touch bootstrap

Reply:
Yeah I think that is way easier. I'll try that ASAP.